onnx/version_converter/adapters/adapter.h

```diff
@@ -13,12 +13,34 @@ class Adapter {
     OpSetID initial_version_;
     OpSetID target_version_;
 
+  protected:
+    bool c2_broadcastable(int64_t axis,
```

Comment:
Rename c2_broadcastable to onnx_opset1_broadcastable.

And make them as functions not member functions.